lib/python/rosie/ws.py

```diff
@@ -61,35 +76,89 @@ def __init__(self, *args, **kwargs):
                 ResourceLocator.default().get_util_home(
                     "lib", "html", "template", "rosie-disco")))
         db_url_map = {}
-        for key, node in rose_conf.get(["rosie-db"]).value.items():
+        for key, node in list(rose_conf.get(["rosie-db"]).value.items()):
```

Comment:
2to3 somewhat over-cautiously wraps every `items` call inside a `list`, it's only actually necessary if we change the keys of the dictionary (i.e. `pop`, `del`, `dict[newkey]=`), otherwise it's safe to remove these.

Reply:
Thanks for the explanation, it should be useful going forward. I wasn't being particular about the details of the Python 3 conversion at this stage of the WIP (for example a lot of blank lines have been blitzed from ``config.py`` which I need to eventually get re-added) as long as changes were vaguely sensible since I was going to do a 'tidy up' once the Disco utility is fully functional & tested under Tornado, particularly since I need to do more 2 to 3 on the tests.